PeerInfo.Lag is wrong type

darkwatchuk opened the pull request:
Changed PeerInfo.Lag to ulong to match GOLANG version.

darkwatchuk committed:
Make the parameter-less StreamConfig constructor public

darkwatchuk committed:
Merge branch 'nats-io:main' into main

darkwatchuk committed:
Merge branch 'nats-io:main' into main

darkwatchuk committed:
Merge branch 'nats-io:main' into main

darkwatchuk committed:
Make NatsMsg<T>.Build Public

mtmk committed:
Add docs

mtmk committed:
Add docs

[no ci]

darkwatchuk committed:
Merge branch 'nats-io:main' into main

darkwatchuk committed:
Merge branch 'nats-io:main' into main

darkwatchuk committed:
Merge branch 'nats-io:main' into main

darkwatchuk committed:
Merge branch 'nats-io:main' into main

darkwatchuk committed:
Changed Peer.Lag to ulong

mtmk reviewed (approved):
LGTM thanks @darkwatchuk

darkwatchuk committed:
Fix data types,

AccountLimits    int -> long

 .MaxMemory
 .MaxStorage
 .MemoryMaxStreamBytes
 .StorageMaxStreamBytes

ApiStats    int -> ulong
 .Total
 .Errors

darkwatchuk commented:
Patched similar in the following according to the GOLANG version.

AccountLimits    int -> long

 .MaxMemory
 .MaxStorage
 .MemoryMaxStreamBytes
 .StorageMaxStreamBytes


ApiStats    int -> ulong
 .Total
 .Errors
